### PR TITLE
Add election participants counter

### PR DESCRIPTION
**Pull request by sparkplug0025**
image

### CODE REVIEW

File: libraries/debug_eos_vm/include/debug_eos_vm/dwarf.hpp [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 #include <eosio/stream.hpp>
 #include <memory>
+#include <optional>
```
Comment (sparkplug0025) [inline libraries/debug_eos_vm/include/debug_eos_vm/dwarf.hpp:5]: @tbfleming without this line clang was failing. is it fine to add this change?

Comment (tbfleming) [reply libraries/debug_eos_vm/include/debug_eos_vm/dwarf.hpp:5]: yes

File: contracts/eden/src/eden-micro-chain.cpp [hunk 2]
```diff
@@ -157,6 +157,7 @@ struct status
    std::string memo;
    eosio::block_timestamp nextElection;
    uint16_t electionThreshold = 0;
+   uint16_t electionParticipants = 0;
```
Comment (tbfleming) [inline contracts/eden/src/eden-micro-chain.cpp:160]: Please use `numElectionParticipants`. This allows us to add `electionParticipants` in the future if needed to list the participants.

Comment (sparkplug0025) [reply contracts/eden/src/eden-micro-chain.cpp]: done.

**Review by tbfleming — APPROVED** [review]

**Pull request merged**